db/migrate/20141206211151_create_scaptimony_assets_policies.rb

```diff
@@ -1,9 +0,0 @@
-class CreateScaptimonyAssetsPolicies < ActiveRecord::Migration
-  def change
```

Comment:
Please always add reverting migration, if I just update scaptimony, how should I know to cleanup db? I think you can take the liberty of not providing automatic data converting script since we don't use it for production but migration is easy to add and makes others life easier.